Shared/Network/NetworkCalls.swift

```diff
@@ -46,6 +49,83 @@ extension NetworkCalls {
             .receive(on: RunLoop.main)
             .eraseToAnyPublisher()
     }
+    
+    func getTimezones(location: Location) -> AnyPublisher<TimezoneResult, Never> {
+        
+        let parameters: [String: Any]? = nil
+        
+        var timezoneUrlString = URLComponents(string: "https://timezone.abstractapi.com/v1/current_time")!
+
+        timezoneUrlString.queryItems = [
+            URLQueryItem(name: "api_key", value: "529f043d73d444649caa77483b0bfd52"),
+            URLQueryItem(name: "location", value: location.description())
+        ]
+        
+        
+        return NetworkService(urlString: timezoneUrlString.string!, httpMethod: .get, parameters: parameters).load()
```

Comment:
can we safe unwrap this?

Reply:
I've been trying to unwrap with `guard`, but can't quite get the right return type.  My code below gives me `AnyPublisher<TimezoneResult, StandardError>`.  I've tried various different ways to transform failure type to `Never` without success.  How do I do this so that the guard check will return a publisher with a `StandardError.defaultValuewithInvalidToken error`?  

```
guard timezoneUrl != nil, let timezoneUrlString = timezoneUrl?.string else {

            return Fail<TimezoneResult, StandardError>(error: StandardError.defaultValueWithInValidToken)
                .eraseToAnyPublisher()
        }
```